CruCentralCoast/Events/EventDetailsVC.swift

```diff
@@ -33,7 +33,7 @@ class EventDetailsVC: UIViewController {
         self.titleLabel.text = self.event?.title
         self.dateLabel.text = self.event?.startDate.toString(dateStyle: .medium, timeStyle: .none).uppercased()
         self.descriptionLabel.text = self.event?.summary
-        self.locationButton.setTitle(self.event?.location?.street, for: .normal)
+        self.locationButton.setTitle(self.event?.locationString, for: .normal)
```

Comment:
We may have made the same change on our PRs which may lead to merge conflicts. I would merge yours in first, then I will merge mine in.

Reply:
Right now the only location data for the events in Firebase is what people entered into the Google Sheet. However, this is just a string, whereas previously we were using street, state, country, etc and then combining it all into a single string. So... I think yours should be merged first, and then I can make this change in the merge. Otherwise location data still won't show up

Reply:
Ahhh ok sounds good!!